usage/jsgui/src/main/webapp/assets/css/base.css

```diff
@@ -196,8 +196,8 @@ li.text-filter input {
     width: 10em;
     margin-top: 3px;
     /* taken from datatables_filter input */    
-    background-image: url("../img/magnifying-glass-right.png");
-    background-size:12px 12px;
+    background-image: url("../img/magnifying-glass-right-icon.png");
+    /* background-size:12px 12px; */
```

Comment:
Decided to keep the properties uncommented to preserve backwards compatibility instead. Just in case someone has customized the search icon.